apps/scratch/src/utilities/student-activity-tracking.ts

```diff
@@ -1,61 +1,76 @@
 import { CrtContextValue } from "../contexts/CrtContext";
 import { isBlockPartOfLargeStack } from "./scratch-block-stack-utils";
-import type { Block, Workspace } from "scratch-blocks";
+import type { Block } from "scratch-blocks";
 import type { WorkspaceChangeEvent } from "../types/scratch-workspace";
 interface TrackMoveParams {
-  workspace: Workspace;
-  blockId?: string;
-  sendRequest?: CrtContextValue["sendRequest"];
-  action?: StudentAction;
+  block: Block;
+  sendRequest: CrtContextValue["sendRequest"];
+  action: StudentAction;
+  solution: Blob;
 }
 
-export type StudentAction = "create" | "move";
+export enum StudentAction {
+  CREATE = "create",
+  MOVE = "move",
+  DELETE = "delete",
```

Comment:
Detail but the convention would to be to use `PascalCase` or `camelCase` for enums, not `ALLCAPS` :)